packages/hardhat-plugin/test/fixture-projects/with-config/hardhat.config.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
   ignition: {
     maxRetries: 1,
     gasIncrementPerRetry: BigNumber.from(1000),
+    pollingInterval: 4,
```

Comment:
Lets show the default of 300. Just in case someone thinks its like 4 seconds.

Reply:
I was intentionally not using the default here so I could test that the user value was being set and properly overwriting the default. The default is written in the docs section for the config where the other default values are listed as well. 

If you'd still like it changed here I can do it, but that's the reasoning for why it's not currently that way.

Reply:
nevermind, i just realized i forgot to push the doc changes i added, so take the above comment as what will soon be the case

Reply:
Ah that makes complete sense. Ignore my request.